docs/reference/search/search-your-data/knn-search.asciidoc

```diff
@@ -242,6 +242,114 @@ POST byte-image-index/_search
 // TEST[s/"k": 10/"k": 3/]
 // TEST[s/"num_candidates": 100/"num_candidates": 3/]
 
+[discrete]
+[[knn-search-quantized-example]]
+==== Byte quantized kNN search
+
+If you want to provide `float` vectors, but want the memory savings of `byte` vectors, you can use the
+<<dense-vector-quantization, quantization>> feature. Quantization allows you to provide `float` vectors, but
+internally they are indexed as `byte` vectors. Additionally, the original `float` vectors are still retained
+in the index. This allows you to use the original `float` vectors for re-scoring, but the `byte` vectors for
+indexing.
+
+To use quantization, you must provide a `quantization_params` object in the `dense_vector` mapping.
```

Comment:
It should refer to `quantization_options`?

Reply:
yep!